src/redux/categories/categories.js

```diff
@@ -0,0 +1,16 @@
+const ACTIVE = 'ACTIVE';
+const NOT_ACTIVE = 'NOT_ACTIVE';
```

Comment:
- [ ] Please, kindly use the duck's pattern as stated in the project requirements.  Also, I recommend giving it a more descriptive name like "CHECK_STATUS" and updating that name on your reducer and action creator.  An example of duck's pattern, with the action creator and the reducer updated:

```javascript
const CHECK_STATUS = bookstore/categories/CHECK_STATUS

const createActionForCategory = () => ({ type: CHECK_STATUS });


const reducerForCategory = (state = [], action) => {
  switch (action.type) {
    case CHECK_STATUS:
      return 'Under construction';
    default:
      return state;
  }
};
```

That way, we will meet this requirement 💪 

- [ ] The "NOT_ACTIVE" is not needed, you can remove that one.